src/packet_analysis/Manager.cc

```diff
@@ -0,0 +1,176 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "Manager.h"
+
+#include "Analyzer.h"
+#include "Dispatcher.h"
+
+using namespace zeek::packet_analysis;
+
+Manager::Manager()
+	: plugin::ComponentManager<packet_analysis::Tag, packet_analysis::Component>("PacketAnalyzer", "Tag")
+	{
+	}
+
+void Manager::InitPostScript()
+	{
+	// Instantiate objects for all available analyzers
+	for ( const auto& analyzerComponent : GetComponents() )
+		{
+		if ( AnalyzerPtr newAnalyzer = InstantiateAnalyzer(analyzerComponent->Tag()) )
+			analyzers.emplace(analyzerComponent->Name(), newAnalyzer);
+		}
+
+	// Read in analyzer map and create dispatchers
+	auto& analyzer_mapping = zeek::id::find("PacketAnalyzer::config_map");
+	if ( ! analyzer_mapping )
+		return;
+
+	auto mapping_val = analyzer_mapping->GetVal()->AsVectorVal();
+	if ( mapping_val->Size() == 0 )
+		return;
+
+	for (unsigned int i = 0; i < mapping_val->Size(); i++)
+		{
+		auto* rv = mapping_val->At(i)->AsRecordVal();
+		//TODO: Make that field a string for usability reasons
```

Comment:
Address TODOs, or can they be removed?

Reply:
See comment above: Do we want to switch back from Tags to strings?